docs/tutorial/getting_started_with_model.rst

```diff
@@ -0,0 +1,156 @@
+Getting started with the Model class
+====================================
+
+This tutorial covers opening an existing RESQML dataset and identifying the high level objects contained within it.
+
+Contents
+1 Prerequisites
+2 Importing the model module
+3 Opening an existing RESQML dataset
+4 Keys to the RESQML high level objects
+5 Selectively listing high level objects
+6 Finding a single high level object
+7 Other methods in the Model class
+8 Summary
+
+Prerequisites
+-------------
+You will need to have resqpy installed in your python environment, along with its dependencies, before proceeding
+
+You will also need an example RESQML dataset (some are available within the resqpy repository). The RESQML dataset will consist of two files, one with extension .epc and the other .h5. This pair of files should have the same name prior to the extension and be located in the same directory (folder). You can use any dataset for this exercise – the detailed output from each step will vary depending on the data.
+
+If you don't have a RESQML dataset, you can generate one by following the instructions in the next tutorial: Creating a Test Dataset
+
+Note: Example file names shown here and in other resqpy tutorials are for a unix environment. If you are working in a Windows environment, the file paths would be in the corresponding format.
+
+Importing the **model** module
+------------------------------
+In this tutorial, we will be using the ``Model`` class which is contained in the model.py module. This can be imported with:
```

Comment:
When mentioning particular resqpy classes, I _think_ we should be able to link directly to the class API documentation with a link like this:

```suggestion
In this tutorial, we will be using the :class:`resqpy.model.Model` class which is contained in the model.py module. This can be imported with:
```

Reply:
yes lets do that